packages/dx-react-scheduler/docs/guides/views.md

```diff
@@ -0,0 +1,33 @@
+# React Scheduler - Views
+
+The Scheduler component provides several types of views. Views display information in various ways, you can choose the most convenient view for your data source or combine few views and switching between them.
+
+## Related Plugins
+
+The following plugins render corresponding views:
+
+- [DayView](../reference/day-view.md) - renders a day layout
+- [WeekView](../reference/week-view.md) - renders a week layout
+- [MonthView](../reference/month-view.md) - renders a month layout
+
+## DayView
+
+The `DayView` displays appointments for a specified date. The Scheduler component arranges appointments from top to bottom. If their time intervals overlap, their width is decreased and they are placed next to each other.
+
+By default the duration of time scale is 24 hours. To change this value define the view plugin's properties `startDayHour` and `endDayHour`. In the following example a time scale starts at 9 a.m. and ends at 7 p.m.
```

Comment:
I think we should define a concept of a time scale somewhere - it's common for all views

Reply:
The time scale describes on the image)

Reply:
I can insert a reference to Fundamentals - Overview. The information about the time scale will be there.